lib/presentation/features/heroes/bloc/heroes_bloc.dart

```diff
@@ -7,30 +7,59 @@ import 'package:marvel/presentation/convertors/convertor.dart';
 import 'package:marvel/presentation/view_models/view_data_character.dart';
 
 class HeroesBloc extends Bloc<HeroesEvent, HeroesState> {
-  final GetCharactersUseCase getCharacters;
+  final GetCharactersUseCase getCharactersUseCase;
+  final int _offset = 0;
```

Comment:
If you do not change this offset - you can make it as default value for `getCharactersUseCase` if offset is not present.